app/views/outputs/new.html.erb

```diff
@@ -0,0 +1,174 @@
+<% content_for :page_title, "Add dissemination or impact" %>
+<% content_for :page_title, truncate(@study.title, length: 50) %>
+
+<% @publication ||= Publication.new %>
+<% @dissemination ||= Dissemination.new %>
+<% @study_impacts = {} if @study_impacts.nil? %>
+
+<div class="site-main site-main--impacts">
+    <div class="container">
+
+        <h4 class="add-outputs__title">Add dissemination or impact for:</h4>
+        <div class="add-outputs__study">
+            <div class="study-header">
+                <%= render partial: "studies/header_primary" %>
+            </div>
+        </div>
+
+        <a class="add-outputs__add-study"><strong>Add another study</strong> sharing the same impacts</a>
+
+      <%= form_tag study_outputs_create_path(@study),
+               class: "add-outputs__form action-sheet" do %>
+        <div class="output-type-accordion">
+
+          <input type="radio"
+                 name="output_type"
+                 id="output-type-publication"
+                 class="output-type-input houdini-input houdini-input--visible"
+                 value="publication"
+                 <%= "checked" if @publication.errors.any?%>>
+          <label for="output-type-publication" class="output-type-label houdini-label">
+              Publication
+          </label>
+
+          <div class="action-sheet__section action-sheet__section--child houdini-target">
+              <%= render partial: "shared/form_errors", locals: { resource: @publication } %>
+              <div class="row">
+                  <div class="col-sm-6">
+                      <%= label :publication, :doi_number, "DOI" %>
+                      <%= text_field :publication, :doi_number, { class: "form-control" } %>
+                      <%= label :publication,
+                                :details,
+                                raw("Or enter details manually&hellip;"),
```

Comment:
If you submit the form with errors, it'd be nice for it to be redisplayed with the manual details already expanded